ButtonMadness/TestWindowController.cs

```diff
@@ -158,17 +158,17 @@ public override void AwakeFromNib ()
 			nibBasedSegControl.SetMenu (buttonMenu, 0);
 
 			// add icons to each segment (applied to both nib-based and code-based)
-			NSImage segmentIcon1 = NSWorkspace.SharedWorkspace.IconForFileType(HFSTypeCode.ComputerIcon);
+			NSImage segmentIcon1 = NSWorkspace.SharedWorkspace.IconForFileType (HfsTypeCode.ComputerIcon);
```

Comment:
were there `[Obsolete]` warnings ?
otherwise it smells like breaking changes...

Reply:
@spouliot there were no `Obsolete` warnings when building this sample, I had just assumed that this was a typo. How would you suggest we check to see if this enum type was generated with a different casing in the past?